### PR TITLE
W-13095610-japaneseuiswitch-se

### PR DESCRIPTION
**Pull request by sebermuley**
https://docs.mulesoft.com/rpa-manager/usermanagement-manage

- Added language switch to _Create a User_
- Added _Change Your Personal Data_ 
- Added some clarifications

### CODE REVIEW

**Comment by sebermuley** [comment]
SME review done

File: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc [hunk 1]
```diff
@@ -195,6 +201,17 @@ Edit users to change their data or permissions:
 For an explanation of the properties, see <<form-create-user, *Create a new User*>>.
 . Click *Save*.
 
+== Edit Your Personal Data
+
+Edit your profile to change your personal data:
+
+. In the navigation panel, click on your user name.
```
[the remark this answers is not in the document]
Comment (hannanelson) [reply rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc:208]: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22click%22

File: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc [hunk 2]
```diff
@@ -195,6 +201,17 @@ Edit users to change their data or permissions:
...
+. Select *Profile* from the drop-down.
```
Comment (hannanelson) [inline rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc:209]:
```suggestion
. Select *Profile* from the drop-down menu.
```

Comment (hannanelson) [reply rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc:209]: (See the "Term Presentation Reference")

File: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc [hunk 3]
```diff
@@ -195,6 +201,17 @@ Edit users to change their data or permissions:
...
+. Edit your data.
+. Click *Save*.
+
+Ask an administrator to assign you additional privileges you need.
```
Comment (hannanelson) [inline rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc:213]:
```suggestion
Ask an administrator to assign you any additional privileges you need.
```

**Review by hannanelson — DISMISSED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request closed without merging**